packages/caliper-cli/lib/bind/bind.js

```diff
@@ -34,31 +36,46 @@ class Bind {
         let sdk = ConfigUtil.get(ConfigUtil.keys.Bind.Sdk, undefined);
         let cwd = ConfigUtil.get(ConfigUtil.keys.Bind.Cwd, undefined);
         let userArgs = ConfigUtil.get(ConfigUtil.keys.Bind.Args, undefined);
+        let file = ConfigUtil.get(ConfigUtil.keys.Bind.File, undefined);
+
+        let bindOptions;
+        const defaultBindOpts = CaliperUtils.parseYaml(path.join(__dirname, './config.yaml'));
 
-        let bindOptions = CaliperUtils.parseYaml(path.join(__dirname, './config.yaml'));
+        if (file) {
+            // User has passed a configuration file to bind
+            if (!fs.existsSync(file)) {
+                let msg = `Passed bind file ${file} does not exist`;
+                logger.error(msg);
+                throw new Error(msg);
+            } else {
+                bindOptions = CaliperUtils.parseYaml(file);
+            }
+        } else {
+            bindOptions = defaultBindOpts;
+        }
 
         let sutList = Object.keys(bindOptions.sut);
         if (!sut) {
-            let msg = `SUT name is not specified. Available SUTs: ${sutList.join(' | ')}`;
+            let msg = `SUT name is not specified. Available SUTs: ${Object.keys(defaultBindOpts.sut).join(' | ')}`;
```

Comment:
Shouldn't we list the SUT names available in the set config file (not necessarily the default)?
So `${Object.keys(bindOptions.sut).join(' | ')}` or simply ``${sutList.join(' | ')}``
Same for the other error messages.

Reply:
this was an attempt to try and protect us from someone passing in an incorrect file - if it is completely wrong, then we won't be able to list anything.

Which raises the question on the need for a validator here

Reply:
Yeah, a validator is definitely needed. Then let's leave it as it is for now.